buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,30 @@
+object Versions {
+    const val appCompat = "1.1.0"
+    const val ktor = "1.6.4"
+    const val coroutines = "1.5.2"
+    const val compose = "1.0.5"
+    const val activityCompose = "1.3.1"
+}
+
+object Dependencies {
```

Comment:
Is there a way we can centralise the dependency versions of the root project and the submodules together?

Reply:
As long as it isn't inside the `buildScript {}` block in the root `build.gradle.kts`: yes.

I haven't included everything in this file *yet*, to make it easier to see what has been migrated and what has not.